test/services/common/translator/base_test_translator.py

```diff
@@ -58,6 +58,9 @@ def _remove_unsent_references(cim: T):
     if isinstance(cim, ConnectivityNode):
         cim.clear_terminals()
 
+    if isinstance(cim, LvFeeder):
```

Comment:
todo: The check for `Feeder` and `LvFeeder` to call `clear_current_equipment()` can be merged into the check for `EquipmentContainer`